Datadog/Datadog.xcodeproj/xcshareddata/xcschemes/DatadogRUM iOS.xcscheme

```diff
@@ -29,8 +29,7 @@
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
          <TestableReference
-            skipped = "NO"
-            parallelizable = "YES">
+            skipped = "NO">
```

Comment:
errr - means longer execution time in CI?

Reply:
This was on purpose as running tests with this scheme in local very rarely succeeds (it mostly hangs on Simulator launch). It works fine without parallel testing. We don't have this option enabled in any other scheme.

This scheme is not used by CI. It uses [`DatadogCore iOS.xscheme`](https://github.com/DataDog/dd-sdk-ios/blob/d51149cac0560bf947e46dd9b472dba9b97c532c/Datadog/Datadog.xcodeproj/xcshareddata/xcschemes/DatadogCore%20iOS.xcscheme#L230-L239) instead, which includes all targets.

I can revert to keep this PR clean, but disabling this option is the first thing I do each time 😉.